packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.1.0]
+### Added
+- Capture token URL from OpenSea API under `tokenURI` property ([#1577](https://github.com/MetaMask/core/pull/1577))
```

Comment:
The main purpose of this PR seems to be to enable NFT usage without the IPFS gateway enabled. It adds a fallback case for when we can't fetch NFT metadata. That seems worth capturing here as well.

I'm not sure the current description is accurate in all cases either. The token URL can come directly from the contract, it doesn't always come from the OpenSea API.

Reply:
I fixed the description in 5ec1573985e08b6ffd1d7c02cd4d08d4ec5cbceb. I'll send you a DM about the other change.

Reply:
Updated in [2be1a3d](https://github.com/MetaMask/core/pull/1634/commits/2be1a3dc902bf8d1b2f3b472fbd329ff42f34879).